frida_tools/tracer.py

```diff
@@ -345,15 +345,39 @@ def _create_resolver_script(self):
     return workingSet;
 }
 
+function parseExportsFunctionPattern (pattern) {
+    var m;
+    var f;
+    var res = pattern.split ('!');
+    
+    if (res.length == 1) {
+        m = '*';
+        f = res[0];
+    }
+    else {
```

Comment:
Style we're using here is: `} else {`

Reply:
Hi, just noticed your kind comment about loving this feature -- thank you! :-)